src/components/Message/MessageText.tsx

```diff
@@ -112,6 +114,8 @@ const UnMemoizedMessageTextComponent = <
         onMouseOver={onMentionsHoverMessage}
       >
         {message.quoted_message && <QuotedMessage />}
+        {replyIsAttachment && <Attachment attachments={replyIsAttachment} />}
```

Comment:
this variable feels like it should be a boolean, how about we do the following and get rid of `replyIsAttachment`...

```suggestion
        {message.attachments?.length ? <Attachment attachments={message.attachments} /> : null}
```